src/Time/Units.hs

```diff
@@ -100,6 +114,41 @@ instance KnownSymbol (ShowUnit unit) => Show (Time unit) where
                                                 n -> '/' : show n
                       in numeratorStr ++ denominatorStr ++ symbolVal (Proxy @(ShowUnit unit))
 
+----------------------------------------------------------------------------
+-- Creation helpers
+----------------------------------------------------------------------------
+
+sec :: Natural -> Second
+sec n = Time (n % 1)
+
+ms :: Natural -> MilliSecond
+ms n = Time (n % 1)
+
+mcs :: Natural -> MicroSecond
+mcs n = Time (n % 1)
```

Comment:
I think it's a good idea to add `{-# INLINE mcs #-}` after every function (replaces `mcs` with proper name` and also to `time` function. Just a suggestions to compiler.